Formula/perl.rb

```diff
@@ -16,6 +16,13 @@ class Perl < Formula
   # Prevent site_perl directories from being removed
   skip_clean "lib/perl5/site_perl"
 
+  patch do
+    # Enable build support on macOS 11.x
+    # https://github.com/Perl/perl5/pull/17946
+    url "https://github.com/Perl/perl5/pull/17946.diff?full_index=1"
+    sha256 "677a6e63412a4cfd2dc1fe16ccca53d789be5ac1acfb5ebb3130c40b51e5e94a"
```

Comment:
Please either use a commit URL or submit this to formula-patches. Note that the `sha256` hash is already mismatched as you've pushed additional commits to that branch.

Reply:
@jonchang Unless it's urgent we can wait until the upstream PR is merged in and use that as the patch, otherwise it just means updating the formula again. I'm pushing changes to the upstream PR shortly so hopefully it will be good enough to get merged in to resolve the issue for now.

Reply:
Just pushed changes so let's hold for a few days to see what happens. If this doesn't get merged soon I'll just create a formula-patch as it will be easier to manage 👍